src/Illuminate/Support/Collection.php

```diff
@@ -920,22 +920,22 @@ public function slice($offset, $length = null)
         return new static(array_slice($this->items, $offset, $length, true));
     }
 
-     /**
-      * Split a collection into a certain number of groups.
-      *
-      * @param  int $numberOfGroups
-      * @return static
-      */
-     public function split($numberOfGroups)
-     {
-         if ($this->isEmpty()) {
-             return $this;
-         }
-
-         $groupSize = ceil($this->count() / $numberOfGroups);
-
-         return $this->chunk($groupSize);
-     }
+    /**
+     * Split a collection into a certain number of groups.
+     *
+     * @param  int    $numberOfGroups
```

Comment:
When I said a missing space, I really did mean exactly one space was missing. :trollface:

Please put exactly two spaces inbetween `int` and `$`.